### PR TITLE
OSS-add-community-tested-versions

### PR DESCRIPTION
**Pull request by bmckercher123**
### Request from Reza to add a community-tested k8s page

- OSS only
- Casey approved trial, Reza will manage community PRs

### CODE REVIEW

**Comment by frozenprocess** [comment]
@bmckercher123 thank you!

File: calico/getting-started/kubernetes/community-tested.mdx [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+- [OpenStack](../getting-started/openstack/)
+- [RKE](../../getting-started/kubernetes/rancher)
+- [Calico for Windows](../../getting-started/kubernetes/windows-calico/)
+
+## Community-tested versions and platforms
+
+This section is updated by {{prodname}} community members. Please share what versions and platforms work for you. 
+
+- To add an entry, go to the bottom of the page and click, **Edit this page** and submit a PR 
+- If your {{prodname}} setup is not working, the best place to start is the {{prodname}} users' slack channel (calicousers.slack.com)
```
Comment (ctauchen) [inline calico/getting-started/kubernetes/community-tested.mdx]:
```suggestion
- If your {{prodname}} setup is not working, the best place to start is the Calico Users Slack channel (calicousers.slack.com)
```

File: calico/getting-started/kubernetes/community-tested.mdx [hunk 2]
```diff
@@ -0,0 +1,53 @@
+---
+description: Provides community inputs on what versions of Kubernetes and platforms work with Calico.
+---
+
+# Community-tested Kubernetes versions
+
+## About {{prodname}} testing
+
+{{prodname}} is tested each release against the Kubernetes versions listed in [System requirements](../getting-started/kubernetes/system-requirements).
```
Comment (ctauchen) [inline calico/getting-started/kubernetes/community-tested.mdx]:
```suggestion
{{prodname}} is tested each release against the Kubernetes versions listed in [System requirements](system-requirements.mdx). 
```

File: calico/getting-started/kubernetes/community-tested.mdx [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+
+# Community-tested Kubernetes versions
+
+## About {{prodname}} testing
+
+{{prodname}} is tested each release against the Kubernetes versions listed in [System requirements](../getting-started/kubernetes/system-requirements). 
+
+{{prodname}} works on many different platforms. The following are common platforms that {{prodname}} tests with some regularity.  
+
+- [AKS](../../getting-started/kubernetes/managed-public-cloud/aks)
```
Comment (ctauchen) [inline calico/getting-started/kubernetes/community-tested.mdx]:
```suggestion
- [AKS](managed-public-cloud/aks.mdx)
```
With relative file paths for cross references. This applies globally.

**Review by ctauchen — DISMISSED** [review]
A few nits, otherwise LGTM!

I like the concept here.

**Pull request merged**